erpnext/accounts/party.py

```diff
@@ -270,9 +270,13 @@ def get_due_date(posting_date, party_type, party, company):
 		elif credit_days_based_on == "Last Day of the Next Month":
 			due_date = (get_first_day(posting_date, 0, 2) + datetime.timedelta(-1)).strftime("%Y-%m-%d")
 
+	if not due_date:
+		due_date = today()
```

Comment:
Today is not realistic if it's not a POS kind of sale. I think it should be entered manually.